src/features/Courses/AddClass/index.jsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup, ModalDialog, Form, ModalCloseButton, Toast, Col,
+} from '@edx/paragon';
+import { Button, Select } from 'react-paragon-topaz';
+import { logError } from '@edx/frontend-platform/logging';
+
+import { addClass } from 'features/Courses/data/thunks';
+import { fetchInstructorsOptionsData } from 'features/Instructors/data/thunks';
+import { fetchClassesData } from 'features/Classes/data/thunks';
+import { updateCurrentPage } from 'features/Courses/data/slice';
+
+import { initialPage } from 'features/constants';
+
+import 'features/Courses/AddClass/index.scss';
+
+const AddClass = ({ isOpen, onClose, courseInfo }) => {
+  const { classId } = useParams();
```

Comment:
This have changed [here](https://github.com/Pearson-Advance/frontend-app-institution-portal/blob/c1113646952000521739ea81e073cc75747dcb89/src/features/Main/index.jsx#L101) so probably will not work properly.